client/secureclient/src/main/java/com/linkedin/openhouse/client/ssl/JobsApiClientFactory.java

```diff
@@ -50,6 +51,9 @@ public ApiClient createApiClient(@NonNull String baseUrl, String token, String t
       throws MalformedURLException, SSLException {
     WebClient webClient = createWebClient(baseUrl, token, truststoreLocation);
     ApiClient apiClient = new ApiClient(webClient);
+    if (token != null && !token.isEmpty()) {
+      apiClient.addDefaultHeader(HttpHeaders.AUTHORIZATION, String.format("Bearer %s", token));
+    }
```

Comment:
```suggestion
```
ditto?

Reply:
I think it's still worth keeping scope for auth-token support in JobsApiClient, since it's still accessed from external OH managed K8 applications. It's not internal only.

Reply:
Can you point me to the code in jobs service and housetables service where authentication token is intercepted and validated?

Reply:
It's not, removed setting token from JobsClient as well